addons/binding/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/api/UniFiController.java

```diff
@@ -105,6 +107,54 @@ public void logout() throws UniFiException {
         executeRequest(req);
     }
 
+    public UniFiClient[] blockStation(UniFiClient client) throws UniFiException {
```

Comment:
Don't copy paste methods, just create 1 method with a boolean parameter `blocked`.

Reply:
The methods returns here something. But it's not used anywhere. So maybe the return is not necessary. Just make it void.

Reply:
Changed to void and only one function. State controlled via function parameter (boolean)